Js/model.js

```diff
@@ -0,0 +1,104 @@
+const projectBtn = document.querySelectorAll('.projectBtn');
+const body = document.querySelector('body');
+const imagesSrc = ['./images/project1.png', './images/project5.png', './images/project2.png', './images/project4.png', './images/project3.png', './images/project6.png'];
+const techTags = ['HTML/CSS', 'Ruby on Rails', 'JavaScript'];
+const work = document.querySelector('.work');
+projectBtn.forEach((element, index) => {
+  /* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please remove the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :+1: